.circleci/build_win.ps1

```diff
@@ -6,6 +6,11 @@ if ("$Env:FORCE_RELEASE" -Or "$Env:CIRCLE_TAG") {
 	New-Item prerelease.txt -type file
 	Write-Host "Building release version."
 }
+else {
+	$last_commit_timestamp = git log -1 --date=unix --format=%ad HEAD
```

Comment:
```suggestion
	git log -1 --date="format:%Y.%-m.%-d" --format="+ci.%cd" HEAD | out-file -encoding ascii prerelease.txt
```
Even here maybe?

Reply:
I tried to change it now but I'm getting empty output from git. `%-m` and `%-d` (i.e. month/day without the leading zero) do not work on Windows. I get output if I remove the hyphens but then the date won't be identical with other platforms.

Since the Windows build seems to have the correct date now and it's emscripten that's failing the comparison, I think it's better to leave the Windows part as is.

Reply:
Also, on Windows there's no way to export `TZ` just for the duration of the command like on Linux so it wouldn't even be shorter:

```powershell
$env:TZ_BACKUP = $env:TZ
$env:TZ = UTC
TZ=UTC git show --quiet --date="format-local:%Y.%-m.%-d" --format="ci.%cd" | out-file -encoding ascii prerelease.txt
$env:TZ = $env:TZ_BACKUP
```

Reply:
Well, just using "format:" everywhere instead of "format-local:" would have been fine, but if %-m doesn't work, sadly, probably nothing better we can do.

Reply:
I thought about just calling it a day and going with local time but the difference when not showing hours is not easy to spot and I think it could lead to subtle bugs down the line :) I also wonder if we can really assume that all the CI machines are guaranteed to run in the same timezone...

Reply:
``format-local`` is local time - ``format`` is commiter/author time, so that should be the same everywhere, shouldn't it :-)? But yeah, it's not important at all ;-).

Reply:
> `format-local` is local time - `format` is commiter/author's time zone, isn't it :-)?

Actually, I was really confused about the difference between these two. I assumed that the non-local date would naturally be UTC. But that wasn't the case here. Your comment was a big "Aha!" moment for me right now :)

> And especially mute, since the %-m thing still wouldn't work.

I didn't dig too deeply into that but I didn't see the minus described explicitly in git docs and I think it might be specific to `date`. Maybe it's a part of a grey area that's not guaranteed to work but git just does not go out of its way to disable so it shows through?